src/components/About/index.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from "react";
+import styles from "./styles.module.css";
+import Link from "@docusaurus/Link";
+
+export default function Stats() {
+    const [stats, setStats] = useState(null);
+
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const response = await fetch(
+                    "https://raw.githubusercontent.com/conda-forge/by-the-numbers/main/data/live_counts.json"

```

Comment:
This is rendered at build time, right? So the numbers won't be fetched live for every visit, or will they? 🤔

Reply:
Every time the page is refreshed or loaded, fetch will go to the API to retrieve the most up-to-date version of the data.